src/navigation/underline-nav.scss

```
@@ -31,7 +31,9 @@
     }
   }
 
-  &.selected {
+  &.selected, // TODO@14.0.0: remove &.selected
+  &[role=tab][aria-selected=true],
+  &[aria-current]:not([aria-current=false]) {
```

From https://github.com/primer/css/issues/948#issuecomment-542807604:

> In scripting we would `removeAttribute` instead of setting it to `false`.

```suggestion
  &[aria-current] {
```

Hmmm.. it would make the selector more performant. Not that it's the bottleneck, but still. On the other hand, might still be good to keep `:not([aria-current=false])` as a safeguard. 🤷‍♂